tool/utils.ts

```diff
@@ -16,10 +16,15 @@ const fetch = initMakeFetchHappen();
 
 shell.config.fatal = true;
 
-// The current platform's operating system. Throws if the operating system
+// Directory that holds source files.
+const BUILD_PATH = 'build';
+
+export type OSKind = 'linux' | 'macos' | 'windows';
+export type ArchKind = 'ia32' | 'x64' | 'arm64';
+
+// Get the platform's operating system. Throws if the operating system
 // is not supported by Dart Sass Embedded.
-const OS: 'linux' | 'macos' | 'windows' = (() => {
-  const platform = process.env.npm_config_platform || process.platform;
+export function getOS(platform: string): OSKind {
```

Comment:
Consider renaming this to something like `nodePlatformToDartPlatform` to clarify that it translates between the two.

Similarly for `getArch()` below.